sentry/src/main/java/io/sentry/ILogger.java

```diff
@@ -31,7 +31,11 @@
    * @param message The message.
    * @param args the formatting arguments
    */
-  void log(SentryLevel level, Throwable throwable, String message, Object... args);
+  void log(
+      @Nullable SentryLevel level,
+      @Nullable Throwable throwable,
+      @Nullable String message,
```

Comment:
IMO `level` and `message` should be required.
One argument is that if I have a throwable I might now need to pass a message but perhaps run a search on the code base right now and see if we have any of such cases.